holidays/countries/finland.py

```diff
@@ -97,6 +101,71 @@ def _populate_public_holidays(self):
         # Second Day of Christmas.
         self._add_christmas_day_two(tr("Tapaninpäivä"))
 
+    def _populate_unofficial_holidays(self):
+        if self._year >= 2012:
+            # Day of Finnish architecture and design, Birthday of Alvar Aalto.
+            self._add_holiday_feb_3(tr("Arkkitehtuurin ja muotoilun päiviä"))
```

Comment:
This holiday marked as "recommended flag day" [here](https://intermin.fi/en/flag-and-arms/flag-days/2024).

Reply:
Hmm, what's your take, should be these type of holidays also included? Looking at the list you linked to, they are very recently introduced (for instance Tove Jansson Day (9th of August) was introduced in 2020), so while I'm quite sure all the Moomin shops around the country would fly flags on this day (and despite how much people like Tove) I doubt that day will be much different from regular one 😅 
 
Or we could for now stick to [The days on which flying the Finnish flag is an established custom](https://intermin.fi/en/flag-and-arms/flag-flying-days).

Reply:
Figured that we can exclude recommended holidays for now.